plugins/node/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -192,13 +192,17 @@ export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
     });
 
     if (command === undefined) return;
-    const query = Object.keys(command.query ?? command.q ?? command).reduce(
-      (obj, key) => {
-        obj[key] = '?';
-        return obj;
-      },
-      {} as { [key: string]: string }
-    );
+
+    const query = this._config?.enhancedDatabaseReporting
```

Comment:
I think we should add a comment to explain why this is done

Reply:
done